ios/brave-ios/Tests/ClientTests/Resources/scripts/cosmetic-filter-tests.js

```diff
@@ -76,6 +78,18 @@
       if (node.id === 'test-upward-selector') {
         results.upwardSelector = window.getComputedStyle(node).display === 'none'
       }
+
+      if (node.id === 'local-iframe') {
+        const localIframeElements = node.contentDocument.querySelectorAll('[id]');
```

Comment:
Is that selector correct? Shouldn't it be `querySelectorAll('iframe[id="test-local-frame-ad"]')`

Reply:
Both would work (this is unit test, and the test is passing because this code found `test-local-frame-ad` as having `display: none`). Right now only one node/element is added to the iframe, so only the one node gets returned but the test is setup we can add more to tests later if needed. This matches above logic by fetching all selectors with an `id`.